src/fsharp/SignatureConformance.fs

```diff
@@ -236,7 +236,7 @@ type Checker(g, amap, denv, remapInfo: SignatureRepackageInfo, checkingSig) =
                   err(fun(x, y, z) -> FSComp.SR.ValueNotContainedMutabilityGenericParametersDiffer(x, y, z, string mtps, string ntps))
                 elif implValInfo.KindsOfTypars <> sigValInfo.KindsOfTypars then
                   err(FSComp.SR.ValueNotContainedMutabilityGenericParametersAreDifferentKinds)
-                elif not (sigArgInfos.Length <= implArgInfos.Length && List.forall2 (fun x y -> List.length x <= List.length y) sigArgInfos (fst (List.chop sigArgInfos.Length implArgInfos))) then 
+                elif not (sigArgInfos.Length <= implArgInfos.Length && List.forall2 (fun x y -> List.length x <= List.length y) sigArgInfos (fst (List.splitAt sigArgInfos.Length implArgInfos))) then 
```

Comment:
splitat and fst -> could be changed to List.take right?

Reply:
Yep
```fsharp
> let xs = [1..10];;
val xs : int list = [1; 2; 3; 4; 5; 6; 7; 8; 9; 10]

> List.splitAt 3 xs |> fst;;
val it : int list = [1; 2; 3]

> List.take 3 xs;;
val it : int list = [1; 2; 3]
```

Reply:
Done.